src/go/options/configgenerator.go

```diff
@@ -119,6 +122,7 @@ func DefaultConfigGeneratorOptions() ConfigGeneratorOptions {
 		ListenerAddress:                  "0.0.0.0",
 		ListenerPort:                     8080,
 		TokenAgentPort:                   8791,
+		DisableOidcDiscovery:             false,
```

Comment:
How about "EnableOidcDiscovery"?

Reply:
We want to keep it turned on by default and give the option to disable it, right? In this case, a `--disable` flag/option works better.

Reply:
Yes, we want OIDC discovery to be on by default. I don't have strong preference.